StortSpelProjekt/Game/src/main.cpp

```diff
@@ -31,6 +33,11 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
         {
             // Currently no scene set, hence the m_pRenderer should not be working.
 
+            if (DEVELOPERMODE_DEVINTERFACE == true)
+            {
+                ImGuiHandler::GetInstance().NewFrame();
+            }
```

Comment:
Is this specific to DX12? If so, maybe put this inside renderers update function?